documentation/Geopotential.tex

```diff
@@ -270,5 +270,78 @@ \subsection*{Singularities}
 \[
 {\mathfrak B\of \gb}{\grad \mathfrak L\of \gl}_{m = 0} = 0
 \]
+\subsection*{Implementation considerations}
+A naïve implementation of the preceding formul{\ae} yields poor performance because of the computation of trigonometric functions, powers and polynomials.  It is advantageous
```

Comment:
You can just write `formulæ` in the source code, that's more readable and XeTeX knows about Unicode.